.golangci.yml

```diff
@@ -12,3 +12,9 @@ linters:
     - ineffassign
     - unconvert
     - unparam
+output:
+  format: github-actions
+issues:
+  exclude:
```

Comment:
@mksha can you describe what these are for?

Reply:
i saw in our github actions we are setting using cli arguments , but i think we can just set them in config file directly. @jgwest

Reply:
I see, thanks @mksha, can you remove the new excludes, I don't think those were present before. It seems like your PR still compiles without them.

Reply:
@jgwest  i have not added anything new, its what we have in our ci https://github.com/argoproj-labs/applicationset/blob/master/.github/workflows/ci-build.yaml#L81

Reply:
Ah, sorry about that @mksha, I forgot we had that and when I did `ripgrep` to find references, it ignores the `.github` folder :thinking: